transformations/dyslexia_words_swap/transformation.py

```diff
@@ -0,0 +1,39 @@
+import os
+import json
+import spacy
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class DyslexiaWordsSwap(SentenceOperation):
+
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+
+    def __init__(self):
```

Comment:
I believe the constructor should also take 'max_outputs' as one of the params, based on which generate function should return perturbed sentences.

Reply:
Done. (even if we cannot generate more than one output, at least it will be consistent with the other constructors)